2.3.2/bullseye/Dockerfile

```diff
@@ -110,28 +111,25 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-ARG VERSION="2.3.2-2"
-ARG SUBMODULES="async,azure,amazon,elasticsearch,google,password,cncf.kubernetes,mysql,postgres,redis,slack,ssh,statsd,virtualenv"
-ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
 ARG AIRFLOW_VERSION="2.3.2"
-ARG ASTRONOMER_FAB_SECURITY_MANAGER_VERSION="1.9.2"
+# ASTRONOMER_CERTIFIED_VERSION is the version in setup.py at airflow-private repo
+ARG ASTRONOMER_CERTIFIED_VERSION="2.3.2+astro.2"
+ENV ASTRONOMER_CERTIFIED_VERSION=${ASTRONOMER_CERTIFIED_VERSION}
+ARG SUBMODULES="async,azure,amazon,elasticsearch,google,password,cncf.kubernetes,mysql,postgres,redis,slack,ssh,statsd,virtualenv"
+ARG AIRFLOW_WHEEL="apache_airflow-${ASTRONOMER_CERTIFIED_VERSION}-py3-none-any.whl"
 
 # Make pip look at our pip repo too, and force it to install these specific
 # versions when ever it installs a module.
 COPY include/pip.conf /etc/pip.conf
 COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
 
-# This constraints will be only used during docker-build
-# and won't be included in the final image
-# It is useful to install packages that are known to work but still allow users to
-# override versions (as opposed to entries in astronomer-pip-constraints.txt)
-COPY build-time-pip-constraints.txt /tmp/build-time-pip-constraints.txt
-
 # Pip install airflow and astro security manager
-RUN pip install "${AIRFLOW_MODULE}" celery flower \
-        --constraint /tmp/build-time-pip-constraints.txt \
-    && pip install "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.5/astronomer_airflow_scripts-0.0.5-py3-none-any.whl" \
-    && pip install "astronomer-fab-security-manager==${ASTRONOMER_FAB_SECURITY_MANAGER_VERSION}"
+COPY airflow_wheels/ /tmp/airflow_wheels
+COPY ac_wheels/ /tmp/ac_wheels
+RUN pip install /tmp/airflow_wheels/${AIRFLOW_WHEEL}[$SUBMODULES] celery flower \
+     --constraint https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_MAJOR_MINOR_VERSION}.txt \
```

Comment:
We shouldn't be using the OSS contraints here -- we want to keep contorl over this ourselves.